### ISSUE TITLE
PDF reporting eventually fails on the Office365 module

### ISSUE THREAD

**Issue by AlexRuiz7** [wazuh/wazuh-kibana-app#3935]
| Wazuh | Elastic | Rev  | Security |
| ----- | ------- | ---- | -------- |
| 4.3   | 7.x     | 4301-1 | OpenSearch 1.2.0 |

| Browser |
| ------- |
| Chrome  |

**Description**
Sometimes, the report generation fails on the Office365 module.

**Steps to reproduce**
1. Navigate to 'Office365'
2. Click on 'Generate Report'
3. See the error.

**Expected Result**
1. The report is generated succesfully.

**Actual Result**
```json
{
  "statusCode": 500,
  "error": "Internal Server Error",
  "message": "5029 - Reporting was aborted (Request failed with status code 401)"
}
```

**Screenshots**

**Additional context**
Add any other context about the problem here. Here you can paste log entries too or any other useful information that may help with the issue.

**Comment by Desvelao** [comment]
This could be caused because the token of Wazuh API expired. There is a related issue https://github.com/wazuh/wazuh-kibana-app/issues/3743 and a PR to fix it https://github.com/wazuh/wazuh-kibana-app/pull/3881.

**Comment by asteriscos** [comment]
Tried to replicate the issue with no avail.

![Peek 2022-04-07 12-09](https://user-images.githubusercontent.com/9343732/162176204-6bf90076-6182-4ad9-81e8-9a9ff5a2a4c3.gif)

**Comment by asteriscos** [comment]
This issue will be closed with PR #3881 which controls session token expiration when generating a PDF report.
